src/components/nfts/NFTCardGallery.scss

```diff
@@ -0,0 +1,91 @@
+#card-gallery-container {
+  width: 280px;
+  padding: 15px 15px 24px 15px;
+  background-color: #21383d;
+  border-radius: 5px;
+  box-sizing: border-box;
+  cursor: pointer;
+
+  p {
+    margin: 0;
+  }
+
+  header {
+    display: flex;
+    margin-bottom: 15px;
+    column-gap: 8px;
+    justify-content: center;
+
+    p {
+      font-size: 14px;
+      font-weight: 700;
+      color: #88a2b3;
+    }
+  }
+
+  main {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    min-width: 0;
+  }
+
+  .token-name {
+    display: block;
+    text-align: center;
+    font-size: 18px;
+    font-weight: 700;
+    margin-top: 15px;
+    color: #ffffff;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    white-space: nowrap;
+  }
+
+  .nft-id-container {
+    display: flex;
+    min-width: 0;
+    margin-top: 15px;
+    justify-content: center;
+
+    span {
+      font-size: 14px;
+      font-weight: 500;
+      color: #899fa8;
+    }
+
+    .token-id {
+      margin-left: 4px;
+
+      > span {
+        color: #c95bf0;
```

Comment:
Please, check the colors into [index.scss](https://github.com/CityOfZion/dora/blob/6167155c400e66960685c636bdb39ec37ae02ec5/src/index.css#L52);